SDK.markdown

```diff
@@ -7,6 +7,104 @@ sidebar: true
 
 We provide you with a NPM package for Node.js and Browser environments. Please see our [GitHub page for details](https://github.com/kontist/sdk).
 
+## Getting started
+This is a very short step by step guide to help you getting started and fetch a few transactions with our API and SDK in Node.js. You can do much more with it, please refer to the documentation or [get in touch with us](mailto:developer@kontist.com).
+
+### Requirements
+Please have Node.js + NPM installed. This example is using plain JavaScript for simplicity, but you can use TypeScript as well.
+
+### Setup a new project
+Create a new folder `kontist-connect-example` and open it in your terminal.
+
+Init the project and add `kontist` as an dependency.
+```bash
+npm init -y
+npm install kontist --save
+```
+
+Create a new file `index.js`
+
+### Create a new client
+
+Add the following code to your `index.js`:
+
+```javascript
+const { Client } = require("kontist");
+
+const client = new Client({
+    clientId: "<client id>",
+    clientSecret: "<client secret>",
+    scopes: ["transactions"]
+});
+```
+
+You need to enter a `<client id>` and `<client secret>` here. Create your own client via the [Client Management](https://kontist.dev/client-management/) section.
+Click "Create Client" and enter a name and a secret. You will need to check the "Transactions" scope for this example. Enter some url like "https://example.com/" into the "Redirect URI" as we do not use it in this example.
+
+![Client Management](/assets/client-management.png){:class="img-responsive"}
+
+After you did create the client click on it in the overview in order to reveal its client id.
+
+
+### Login and fetch transactions
+Note: In this example you only want to access our own account. You can do this by using *your own* client and *your* username and password with the `fetchTokenFromCredentials` method. If you want to create an application that other users can use as well, then please follow the authentication section in the documentation. You need to implement the OAuth2 authorization code flow in that case.
+But for our own user this can be simplified by just using username and password like this:
+
```

Comment:
Does the user care for this?
I want to clarify this somewhere to avoid confusion.

Reply:
I think it's a good idea to mention it, but I'm also afraid that this:
> You need to implement the OAuth2 authorization code flow in that case.

Might scare off some potential users less familiar with this whole OAuth2 terminology. Also if they use the built-in sdk methods for it they don't have to actually implement the whole thing 🙂 
I think I would just remove this sentence and keep the rest. 

> then please follow the authentication section in the documentation.

Maybe adding a link to the authentication documentation could be useful as well? e.g. `follow the [authentication section](#authentication section) in the documentation`